.dockerignore

```diff
@@ -13,6 +13,9 @@
 !site_scons
 !.clang-format
 !requirements.txt
+!ci/rpm/install_maldet.sh
+!ci/rpm/build_url_repo.sh
+!ci/rpm/distro_info.sh
```

Comment:
I had a bit of a go at this myself after making this comment and have come up with https://github.com/daos-stack/daos/pull/9898

If you have to re-push this can you move these files to utils/ci and they'll be available to the docker images automatically and correctly trigger the right github actions jobs as well.

Reply:
Now landed.  Can you update this PR to move these new files to utils/ci please where they will be accessible to docker.

Reply:
Will be updated in the next push.